src/main/java/com/github/tomitakussaari/phaas/util/JwtHelper.java

```diff
@@ -1,15 +1,14 @@
 package com.github.tomitakussaari.phaas.util;
 
 import com.github.tomitakussaari.phaas.model.DataProtectionScheme.CryptoData;
-import com.github.tomitakussaari.phaas.user.PhaasUserDetails;
+import com.github.tomitakussaari.phaas.user.PhaasUser;
 import com.nimbusds.jose.*;
 import com.nimbusds.jose.crypto.DirectDecrypter;
 import com.nimbusds.jose.crypto.DirectEncrypter;
 import com.nimbusds.jose.crypto.MACSigner;
 import com.nimbusds.jose.crypto.MACVerifier;
 import com.nimbusds.jwt.JWTClaimsSet.Builder;
 import com.nimbusds.jwt.SignedJWT;
-import lombok.experimental.UtilityClass;
 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.experimental.UtilityClass'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)